src/application/state.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+
+
+class ApplicationState:
+    """
+    Application state.
+
+    For the simplicity, we store this as a global, but better would be to
+    instantiate this as an object and pass at around.
```

Comment:
Perhaps we should do what the comment suggests and instantiate this as an object and pass it around instead of using static methods (?).

Reply:
you're right but that would be through `ThoughtBox` into `ScrollMenuDialog` because `ScrollMenuDialog` needs it.

Reply:
Yes exactly, `ThoughtBox` needs an attribute of type `ApplicationState`